code/game/gamemodes/nuclear/nuclearbomb.dm

```diff
@@ -96,16 +95,17 @@
 			return CINEMATIC_NUKE_FAR
 	return CINEMATIC_NUKE_FAR
 
-/obj/machinery/nuclearbomb/syndicate/Initialize()
-	. = ..()
-	var/obj/machinery/nuclearbomb/existing = locate("syndienuke") in GLOB.nuke_list
```

Comment:
This is how nukeops find the proper code to give, you can't just remove it without replacing it.

Reply:
I don't understand.

Reply:
The proper bomb activation code

Reply:
Yeah, I fixed it and tested. It works now.